contrib/drivers/openstack/cinder/cinder.go

```diff
@@ -170,6 +186,7 @@ func (d *Driver) CreateVolume(req *pb.CreateVolumeOpts) (*model.VolumeSpec, erro
 		Name:        req.GetName(),
 		Description: req.GetDescription(),
 		Size:        int(req.GetSize()),
+		VolumeType:  req.GetPoolName(),
```

Comment:
Can you submit a separate PR to just pass in the VolumeType specified by user?  Talk to @leonwanghui about it.  Thanks.

Reply:
> Can you submit a separate PR to just pass in the VolumeType specified by user? Talk to @leonwanghui about it. Thanks.

OK